src/Reflection/AbstractFunction.php

```diff
@@ -364,64 +349,46 @@ public function __get($key)
      * @param mixed $value
      * @return void
      */
-    public function __set($key, $value)
+    public function __set(string $key, $value): void
     {
         $this->config[$key] = $value;
     }
 
     /**
      * Set method's namespace
      *
-     * @param string $namespace
      * @throws Exception\InvalidArgumentException
-     * @return void
      */
-    public function setNamespace($namespace)
+    public function setNamespace(?string $namespace = null): void
     {
         if (empty($namespace)) {
             $this->namespace = '';
```

Comment:
I see it in the other setter, that you changed the line to `null`, but not above.
Again: we should decide if we want to support null values, if so, probably here we should have `null` instead.

Reply:
Not sure, what you mean here. Can you elaborate?

Reply:
As discussed in chat: Fixed by setting `$namespace` to `null` if new value is empty.